src/App.vue

```diff
@@ -10,9 +10,17 @@
 </template>
 
 <script>
-
 export default {
-  name: 'App'
+  name: 'App',
+  data() {
+    return {
+    }
+  },
+  computed: {
+    campaigns() {
+      return this.$store.state.campaigns
+    }
+  }
```

Comment:
Ooops, don't quite need this at this level since each component can access store. Will delete.

Reply:
You can delete it and push up again to the same branch before we merge